controllers/front/CartController.php

```diff
@@ -133,7 +133,7 @@ public function displayAjaxUpdate()
         $productsInCart = $this->context->cart->getProducts();
         $updatedProducts = array_filter($productsInCart, [$this, 'productInCartMatchesCriteria']);
         $updatedProduct = reset($updatedProducts);
-        $productQuantity = $updatedProduct['quantity'];
+        $productQuantity = $updatedProduct['quantity'] ?? false;
```

Comment:
Shouldn't it be 0? Or null? Why false?

Reply:
I think 0 would be more appropriate, what do you think?

Reply:
Yes, I changed it to 0 :+1: